README.md

```diff
@@ -39,10 +38,37 @@ You can find a list of available interfaces using
 
 `.\geneva-cli.exe list-adapters`
 
+## How to test
+
+You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
+
+You can run `go test` to run some unit tests, currently there is only tests for validating strategies.txt
+
+## How to install as service
+
+Open PowerShell in administrator mode before running any commands
+
+### Install Command, in manual mode
+
+```New-Service -Name "Geneva-CLI" -DisplayName "WinGeneva" -Description "Geneva CLI for Windows" -StartupType Manual -BinaryPathName "Path-To-Geneva.exe --args"```
```

Comment:
Bit of a nit, but I don't think the service itself should be called "Geneva-CLI"; CLI refers to the command-line program (which I understand the service is utilizing).  You could just call it "Geneva".  Same goes for the description.